src/ClassTransformer.ts

```diff
@@ -1,5 +1,6 @@
 import {ClassTransformOptions} from "./ClassTransformOptions";
-import {TransformOperationExecutor, TransformationType} from "./TransformOperationExecutor";
+import {TransformOperationExecutor} from "./TransformOperationExecutor";
+import { TransformationType} from "./TransformationType";
```

Comment:
There is a space inside the brackets here (@NoNameProvided, we will decide and review the lint rules to check those).